### ISSUE TITLE
React Portal update required

### ISSUE THREAD

**Issue by Darper** [DigitalRiver/react-atlas#959]
React has deleted ReactDOM.unstable_renderSubtreeIntoContainer(). It should be replaced with ReactDOM.createPortal() in Portal.js. But doing so causes this response when running tests.

` FAIL  src/Dialog/__tests__/dialog.test.js (5.118s)
  ● Dialog component - Test correct render › encountered a declaration exception

    Invariant Violation: Target container is not a DOM element.

      83 |   "className": PropTypes.oneOfType([
      84 |     PropTypes.string,
    > 85 |     PropTypes.object,
      86 |     PropTypes.array
      87 |   ]),
      88 | 

      at invariant (node_modules/fbjs/lib/invariant.js:42:15)`

**Comment by Darper** [comment]
This is breaking tests and we can't merge in new changes or publish until it is fixed.

**Comment by Darper** [comment]
Removing failing test. Adding a new issue to create a proper snapshot test in the future.
